onnxruntime/test/framework/float_8_test.cc

```diff
@@ -5,6 +5,11 @@
 
 #include <vector>
 
+#ifdef USE_CUDA
+// Needed for CUDA_VERSION check in float8.h
+#include <cuda.h>
```

Comment:
Same here as in float_4_test.cc - We want to include the cuda header on CUDA enabled builds to bring in CUDA_VERSION which is a gating check to use CUDA conversion routines in float_8.h